README.md

```diff
@@ -144,3 +144,45 @@ Execute the following steps in a PowerShell terminal:
 8. Open [Docker Desktop](https://www.docker.com/products/docker-desktop/) and leave it opened in the background.
 
 Now, open a Git Bash terminal and follow the instructions in the "[Run the script](#run-the-script)" section as well as the subsequent sections. You might need to install Microsoft Visual C++ 14.0 or greater.
+
+## Advanced usage
+
+In this chapter, we invite you to dive deeper into our service's capabilities. 
+Tailored for enthusiasts eager to customize and optimize, this section empowers you to make nuanced adjustments, 
+enhancing your experience to match unique requirements.
+
+##### Policy weights
+
+This script automatically sets some default weights to the agent's policy as a warm start 
+to help convergence and improve tool selection. 
+These data were obtained after many days of running the service and are set 
+[here](https://github.com/valory-xyz/trader-quickstart/blob/0f093ebbf0857b8484a017912c3992f00fbe1a29/run_service.sh#L133-L137).
+As a result, the current weights are always deleted and replaced by this strategy 
+which is considered to boost the initial performance of the service.
+
+However, you may have found better performing policy weights and would like to remove this logic. 
+It can easily be done, by removing this method call, 
+[here](https://github.com/valory-xyz/trader-quickstart/blob/0f093ebbf0857b8484a017912c3992f00fbe1a29/run_service.sh#L698), 
+in order to set your own custom warm start. 
+Setting your own custom weights can be done by editing the corresponding files in `.trader_runner`.
+Moreover, you may store your current policy as a backup before editing those files, using the following command:
+
+```shell
+cp ".trader_runner/available_tools_store.json" ".trader_runner/available_tools_store_$(date +"%d-%m-%Y")".json
+```
+
+##### Tool selection
```

Comment:
Step 1 should be to tell the user about the env variable that adjusts the policy punishment

Reply:
We do not have an environment variable for the punishment, this is fixed to the mech's price.

Reply:
Open an issue https://github.com/valory-xyz/trader/issues/120

I suggest we close this comment.